migrations/1661202676736_drop-content-and-title-columns-from-posts-tags-table.js

```diff
@@ -3,7 +3,7 @@
 exports.shorthands = undefined
 
 exports.up = (pgm) => {
-    pgm.dropColumns('posts_tags', ['title', 'content'], { ifExists: true })
+	pgm.dropColumns('posts_tags', ['title', 'content'], { ifExists: true })
```

Comment:
why this change @beblicarl

Reply:
when i opened this file, there was an eslint error that had to do with the tab spacing, so i just corrected the spacing error